inventory/host_vars/staging.openfoodnetwork.org.uk/config.yml

```diff
@@ -7,3 +7,5 @@ rails_env: staging
 admin_email: dev.ofnuk@gmail.com
 
 unicorn_timeout: 120
+
+enable_rails_apm: "true" # has to be explicitly defined as a string due to some datadog bug
```

Comment:
Do we really want to permanently enable this in staging?

Reply:
IMO it's the only reliable way to ensure we don't break production while adding things to Datadog. I think we'll spend much more time checking whether things could break production and possibly fixing it when it's too late than the extra money this costs. 

Besides, monitoring is a too important piece to assume any unnecessary risks.

Reply:
I think this is great. :+1: